js/src/dropdown.js

```diff
@@ -55,6 +64,21 @@ const Dropdown = (($) => {
     VISIBLE_ITEMS : '.dropdown-menu .dropdown-item:not(.disabled)'
   }
 
+  const AttachmentMap = {
+    TOP    : 'top',
```

Comment:
shouldn't be better if default will be 'top-start' and 'bottom-start'.
For my eyes centered dropdowns looks wired :)